README.md

```diff
@@ -1,6 +1,84 @@
-# SendBird Android Sample
+# Sendbird Android samples
+![Platform](https://img.shields.io/badge/platform-ANDROID-orange.svg)
+![Languages](https://img.shields.io/badge/language-JAVA-orange.svg)
 
-The samples in this repository are fully functional messaging applications built using [SendBird](https://sendbird.com) Android SDK.
-  1. **basic:** SendBird android basic sample app without SyncManager. It doesn't support local caching.
-  2. **syncmanager:** Same as Basic sample with SyncManager integrated. It supports local caching.
-  3. **uikit:** Sendbird android sample using Sendbird-uikit sdk
+## Introduction
+
+[Sendbird](https://sendbird.com) provides the chat API and SDK for your app, enabling real-time communication among the users. Here are various samples built using Sendbird Chat SDK.
+
+- [**Chat sample**](#chat-sample) has core chat features. Group channel and open channel are the two main channel types in which you can create various subtypes where users can send and receive messages. This sample is written in Java with [Sendbird Chat SDK](https://github.com/sendbird/SendBird-SDK-Android/tree/master/com/sendbird/sdk/sendbird-android-sdk).
+
+- [**SyncManager sample**](#syncmanager-sample) is equipped with a local cache along with core chat features. For faster data loading and caching, the sample synchronizes with the Sendbird server and saves a list of group channels and the messages within the local cache into your client app. This sample is written in Java with [Sendbird SyncManager](https://github.com/sendbird/sendbird-syncmanager-android) and [Sendbird Chat SDK](https://github.com/sendbird/SendBird-SDK-Android/tree/master/com/sendbird/sdk/sendbird-android-sdk). Find more about SyncManager on [Sendbird SyncManager document](https://docs.sendbird.com/ios/sync_manager_getting_started).
```

Comment:
`[Sendbird SyncManager document](https://docs.sendbird.com/ios/sync_manager_getting_started)` the link leads me to the doc for ios.

Reply:
Done!